run/cache.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+function upload() {
+  echo "::group::Upload to Cache"
+
+  echo "::debug::uploading$UNCACHED"
+
+  if [[ $CACHE =~ ^s3:// ]]; then
+    if [[ $CACHE =~ \? ]]; then
+      CACHE="$CACHE&secret-key=$NIX_KEY_PATH"
+    else
+      CACHE="$CACHE?secret-key=$NIX_KEY_PATH"
+    fi
+  fi
+
+  #shellcheck disable=SC2086
+  nix copy --from "$BUILDER" --to "$CACHE" $UNCACHED
```

Comment:
We might just be able to filter out the action drv here?

Reply:
why do we want to filter the action derivation exactly?

If so we don't need to filter at all, just change the json query to simply pull out all the targetDrvs

Reply:
Is there value in caching it? I may not be seeing the case, but actions are typically a 3 line bash script (that is probably faster built than pulled from the cache).